test/ApiCheck.Test/ApiCheck.Test.csproj

```diff
@@ -3,8 +3,7 @@
   <Import Project="..\..\build\common.props" />
 
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp2.0;net46</TargetFrameworks>
-    <TargetFrameworks Condition="'$(OS)' != 'Windows_NT'">netcoreapp2.0</TargetFrameworks>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
```

Comment:
Keep the .NET Framework tests. It's important that we ensure this tool still runs on .NET Framework.